README.md

```diff
@@ -101,7 +102,9 @@ Importing the Gradle project into Intellij and Eclipse both work.
 
 ## Docker
 
-alf.io can be run for development with Docker Compose:
+Container images are available on https://hub.docker.com/r/alfio/alf.io/tags.
+
+alf.io can also be run locally for development with Docker Compose:
```

Comment:
Or for production - that's how I use it.

Reply:
@asymmetric sure - I've just removed "locally for development"